handler/cache.go

```diff
@@ -0,0 +1,9 @@
+package handler
+
+// Cache for the graphqlHandler
+type Cache interface {
+	// Get looks up a key's value from the cache.
+	Get(key interface{}) (value interface{}, ok bool)
+	// Add adds a value to the cache.  Returns true if an eviction occurred.
+	Add(key, value interface{}) (evicted bool)
```

Comment:
`evicted` is not used

Reply:
its interface  for instead https://github.com/99designs/gqlgen/blob/master/handler/graphql.go#L384

Reply:
It forces other implementations to return a value we don't use